modules/installation-complete-user-infra.adoc

```diff
@@ -147,7 +147,7 @@ If the pod logs display, the Kubernetes API server can communicate with the
 cluster machines.
 
 ifndef::ibm-power[]
-. For an installation with Fibre Channel Protocol (FCP), additional steps are required to enable multipathing. Do not enable multipathing during installation.
+. For an installation with Fibre Channel Protocol (FCP), additional steps are required to enable multipathing.
```

Comment:
In l154 below, we have added a note, "When installing with multipath, it is strongly recommended to enable it at installation time, and not at a later time, which can cause problems." This obviously contradicts the sentence that is being removed in this PR.

Reply:
@bobfuru I've got confirmation that multipathing is not supported with KVM on IBM Z. Therefore, step 3 must be removed for the KVM install docs. 

I can't comment before line 149 so I'll just add everything here. 

- After line 28 insert: 

```
ifeval::["{context}" == "installing-ibm-z-kvm"]
:ibm-z-kvm:
endif::[]

ifeval::["{context}" == "installing-restricted-networks-ibm-z-kvm"]
:ibm-z-kvm:
:restricted:
endif::[]
```

Line 149: s/ifndef::ibm-power,ibm-z[]/ifndef::ibm-power,ibm-z,ibm-z-kvm[]